sample/lit-todo/main.js

```diff
@@ -55,20 +54,20 @@ rep.onSync = syncing => {
 
 rep.subscribe(
   async tx => {
-    return (await tx.scanAll({indexName: 'byOrder'})).map(([_, v]) => v);
+    return (await tx.scanAll({indexName: 'byOrder2'})).map(([_, v]) => v);
```

Comment:
Why order2?

Reply:
Potential API usability issue: we have defined `createIndex` to throw if the named idx already exists but with a different definition, and drop throws if asked to drop and index that *doesn't* exist, and there's no `get`/`has` for index.